app/views/shared/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<% if media.errors.any? %>
+  <h4>The form has <%=pluralize(media.errors.count, "error") %>:</h4>
```

Comment:
Yessss! I love how you handled these form error messages. 👏🏾👏🏾👏🏾
The `pluralize()` is a nice touch.